### PR TITLE
Pass activated rules as an input to OPA

### PR DESCRIPTION
**Pull request by uri-weisman**
This PR modifies the way we pass the benchmarks' activated rules to OPA.
Instead of re-bundling OPA with `data.yml`, we now pass those rules as input for each resource evaluation.

Relates:
- https://github.com/elastic/security-team/issues/4497
- https://github.com/elastic/security-team/issues/4452

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -262,6 +262,7 @@ replace (
 	github.com/apoydence/eachers => github.com/poy/eachers v0.0.0-20181020210610-23942921fe77 //indirect, see https://github.com/elastic/beats/pull/29780 for details.
 	github.com/dop251/goja => github.com/andrewkroh/goja v0.0.0-20190128172624-dd2ac4456e20
 	github.com/dop251/goja_nodejs => github.com/dop251/goja_nodejs v0.0.0-20171011081505-adff31b136e6
+	github.com/elastic/csp-security-policies => github.com/uri-weisman/csp-security-policies v0.0.18-go-lib.0.20220801134428-747e21fc1617
```
Comment (uri-weisman) [inline go.mod:265]: will be replaced after https://github.com/elastic/security-team/issues/4452 is merged.

File: beater/validator.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func (v *validator) Validate(cfg *agentconfig.C) error {
 		return fmt.Errorf("No streams received in reconfiguration %v", cfg.FlattenedKeys())
 	}
 
-	if c.Streams[0].DataYaml == nil {
+	if c.Streams[0].RuntimeCfg == nil {
 		return fmt.Errorf("data_yaml not present in reconfiguration %v", cfg.FlattenedKeys())
```
Comment (yashtewari) [inline beater/validator.go:44]: Should update the error message with the refactored names.

File: config/config.go [hunk 3]
```diff
@@ -22,12 +22,12 @@ package config
...
 	"context"
+	"fmt"
 	awssdk "github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/elastic/beats/v7/libbeat/processors"
 	"github.com/elastic/beats/v7/x-pack/libbeat/common/aws"
 	"github.com/elastic/elastic-agent-libs/config"
 	"github.com/elastic/elastic-agent-libs/logp"
-	"gopkg.in/yaml.v3"
 	"time"
 )
```
Comment (yashtewari) [inline config/config.go:32]: Not really a part of your PR, but do you mind grouping these imports into three groups separated by empty lines: standard library imports, Elastic org imports, third-party imports.
```
import (
	"context"
	"fmt"
	"time"
	
	"github.com/elastic/beats/v7/libbeat/processors"
	"github.com/elastic/beats/v7/x-pack/libbeat/common/aws"
	"github.com/elastic/elastic-agent-libs/config"
	"github.com/elastic/elastic-agent-libs/logp"
	
	awssdk "github.com/aws/aws-sdk-go-v2/aws"
)
```

**Review by yashtewari — APPROVED** [review]
Looks good, I for one am glad to leave `DataYaml` behind!

File: go.mod (hunk 1 above)
Comment (yashtewari) [reply go.mod]: You mean this one? https://github.com/elastic/csp-security-policies/pull/116/files

File: config/config.go (hunk 3 above)
Comment (uri-weisman) [reply config/config.go:32]: Sure thing

**Pull request merged**